modules/pxp-module-puppet

```diff
@@ -282,6 +282,10 @@ def action_run(input)
         1
       end
 
+      # flush the stdout/stderr before writing the exitcode
+      # file to avoid pxp-agent reading incomplete output
+      $stdout.fsync
```

Comment:
No, and there's leatherman in between as well; so, as said, it's for consistency.